src/DssSpell.t.sol

```diff
@@ -557,76 +561,83 @@ contract DssSpellTest is DssSpellTestBase {
+            assertEq(vest.usr(janskyStreamId), janskyAddress, "testVestMKR/invalid-address");
+            assertEq(vest.bgn(janskyStreamId), OCT_01_2023, "testVestMKR/invalid-bgn");
+            assertEq(vest.clf(janskyStreamId), OCT_01_2023, "testVestMKR/invalid-clif");
+            assertEq(vest.fin(janskyStreamId), janskyFin, "testVestMKR/invalid-calculated-fin");
+            assertEq(vest.fin(janskyStreamId), SEP_30_2024, "testVestMKR/invalid-fin-variable");
+            assertEq(vest.mgr(janskyStreamId), address(0), "testVestMKR/invalid-manager");
+            assertEq(vest.res(janskyStreamId), 1, "testVestMKR/invalid-res");
+            assertEq(vest.tot(janskyStreamId), 216 ether, "testVestMKR/invalid-total"); // note: ether is a keyword helper, only MKR is transferred here
+            assertEq(vest.rxd(janskyStreamId), 0, "testVestMKR/invalid-rxd");
+            prevBalance = gov.balanceOf(janskyAddress);
             GodMode.setWard(address(vest), address(this), 1);
-            vest.unrestrict(chronicleStreamId);
-            vm.warp(chronicleFin);
-            vest.vest(chronicleStreamId);
-            assertEq(gov.balanceOf(chronicleAddress), prevBalance + 2_216.4 ether, "testVestMKR/invalid-received-amount");
+            vest.unrestrict(janskyStreamId);
+            vm.warp(janskyFin);
+            vest.vest(janskyStreamId);
+            assertEq(gov.balanceOf(janskyAddress), prevBalance + 216 ether, "testVestMKR/invalid-received-amount");
         }
 
-        { // check JETSTREAM stream
-            address jetstreamAddress = wallets.addr("JETSTREAM");
-            uint256 jetstreamStreamId = prevStreamCount + 2;
-            uint256 jetstreamFin = JUN_26_2023 + 6 days + 366 days + 183 days - 1;
-            assertEq(vest.usr(jetstreamStreamId), jetstreamAddress, "testVestMKR/invalid-address");
-            assertEq(vest.bgn(jetstreamStreamId), JUN_26_2023, "testVestMKR/invalid-bgn");
-            assertEq(vest.clf(jetstreamStreamId), JUN_26_2023, "testVestMKR/invalid-clif");
-            assertEq(vest.fin(jetstreamStreamId), jetstreamFin, "testVestMKR/invalid-calculated-fin");
-            assertEq(vest.fin(jetstreamStreamId), DEC_31_2024, "testVestMKR/invalid-fin-variable");
-            assertEq(vest.mgr(jetstreamStreamId), address(0), "testVestMKR/invalid-manager");
-            assertEq(vest.res(jetstreamStreamId), 1, "testVestMKR/invalid-res");
-            assertEq(vest.tot(jetstreamStreamId), 1_619.93 ether, "testVestMKR/invalid-total"); // note: ether is a keyword helper, only MKR is transferred here
-            assertEq(vest.rxd(jetstreamStreamId), 0, "testVestMKR/invalid-rxd");
-            prevBalance = gov.balanceOf(jetstreamAddress);
+        { // check VOTEWIZARD stream
+            address voteWizardAddress = wallets.addr("VOTEWIZARD");
+            uint256 voteWizardStreamId = prevStreamCount + 2;
+            uint256 voteWizardFin = OCT_01_2023 + 366 days - 1;
```

Comment:
Same as above:
Add a comment explaining we are only using 366 days because there's a leap year adjustment in between.